### PR TITLE
1.6.3

### PR DESCRIPTION
**Pull request by ildarkhasanshin**
- add option for new yandex's counter code
- tested up to wordpress 5.0

### CODE REVIEW

File: readme.txt [hunk 1]
```diff
@@ -1,10 +1,10 @@
 === Yandex Metrica ===
 
-Contributors: m_uysl
+Contributors: m_uysl, [ildar r. khasanshin](https://10021987.ru)
```
[the remark this answers is not in the document]
Comment (ildarkhasanshin) [reply readme.txt]: renamed

File: yandex-metrica.php [hunk 2]
```diff
@@ -247,11 +247,13 @@ private function current_user_has_access( $array ) {
 	public function wp_footer() {
 		if ( ! empty( $this->options['counter_id'] ) ) {
 
+			$tracker_file = $this->options["track-logged-in"] === true ? "tracker-js-new.php" : "tracker-js.php";
```
Comment (mustafauysal) [inline yandex-metrica.php]: checking wrong option, it must be `$this->options["new_yandex_code"]`

File: templates/settings.php [hunk 3]
```diff
@@ -34,7 +34,8 @@
 	$this->options['track-logged-in']     = ( $_POST['track-logged-in'] == "no" ) ? false : true;
 	$this->options["untrack-roles"]       = ! empty( $_POST["tracker_role"] ) ? array_map( 'esc_attr', $_POST["tracker_role"] ) : "";
 	$this->options["widget-access-roles"] = ! empty( $_POST["widget_access"] ) ? array_map( 'esc_attr', $_POST["widget_access"] ) : "";
-	$this->options["tracker-address"] = ! empty( $_POST["tracker-address"] ) ? esc_url_raw( $_POST['tracker-address'] ) : "https://mc.yandex.ru/metrika/watch.js";
+	$this->options['new_yandex_code']     = empty( $_POST['new_yandex_code'] ) ? false : true;
+	$this->options["tracker-address"]     = ! empty( $_POST["tracker-address"] ) ? esc_url_raw( $_POST['tracker-address'] ) : ($this->options['new_yandex_code'] ? "https://mc.yandex.ru/metrika/tag.js" : "https://mc.yandex.ru/metrika/watch.js");
```
Comment (mustafauysal) [inline templates/settings.php]: avoid short if/else controls if there is a complicated situation (better readability)

**Review by mustafauysal — COMMENTED** [review]
notes for feedback, already merged @ildarkhasanshin

**Pull request merged**